routes/api/v1/dashboards.js

```diff
@@ -0,0 +1,69 @@
+const Dashboard = require('../../../models/dashboards.js')
```

Comment:
Between parents and params? So instead of ```router.post('/', (request, response, next) ``` you would like to see ```router.post ('/', (request, response, next) ``` ? Or ```router.post( '/',  ( request, response, next ) ``` ?